### PR TITLE
enable Kuryr SDN on 4.10

### PR DESCRIPTION
**Pull request by masco**

### CODE REVIEW

File: dags/openshift_nightlies/manifest.yaml [hunk 1]
```diff
@@ -97,6 +97,7 @@ platforms:
       releaseStream: 4.10.0-0.nightly
       profiles:
       - default
```
Comment (Harshith-umesh) [inline dags/openshift_nightlies/manifest.yaml:99]: can we rename "default" to openshiftsdn or just sdn? will make it easier to interpret

Comment (masco) [reply dags/openshift_nightlies/manifest.yaml:99]: I see this is the practice we have used in all other clouds

Comment (Harshith-umesh) [reply dags/openshift_nightlies/manifest.yaml:99]: yes but with this new pr: https://github.com/cloud-bulldozer/airflow-kubernetes/pull/79 we're planning on getting rid of the default dag type for aws and azure. when it comes to openstack dags it make even more sense to rename defeault to sdn.
 also with this pr, we can enable different cron schedules for different profiles, do you wan't me to do that for openstack as well?  https://github.com/Harshith-umesh/airflow-kubernetes/blob/b6336e3c014a9526186bafc21a2722614d2ab7dd/dags/openshift_nightlies/manifest.yaml#L16

Comment (masco) [reply dags/openshift_nightlies/manifest.yaml]: done

**Review by Harshith-umesh — APPROVED** [review]
LGTM

File: dags/openshift_nightlies/manifest.yaml (hunk 1 above)
Comment (whitleykeith) [reply dags/openshift_nightlies/manifest.yaml]: @masco to keep in line with the above PR just use `sdn` instead of `openshift-sdn`

Comment (masco) [reply dags/openshift_nightlies/manifest.yaml]: @whitleykeith done

**Pull request merged**